src/styles/blocks/card.css

```diff
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Nice code! I suggest to create a separate normalize file and move this block of code there

Reply:
thanks